<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="songPrint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>1141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADFSDFSDF</string>
    </property>
   </widget>
   <widget class="QComboBox" name="melodyList">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Hymn for the Weekend - Coldplay</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C-BooL - DJ Is Your Second Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Demons - Imagine Dragons</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Медуза - MATRANG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sweet Dreams - Eurythmics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Skillet - The Resistance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Goodbye - Glenn Morisson</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>David Guetta - Titanium ft. Sia</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="inputter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>1141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tw Cen MT'; font-size:22pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ABCD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1215</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAudiobug">
    <property name="title">
     <string>Audiobug</string>
    </property>
   </widget>
   <addaction name="menuAudiobug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
